### PR TITLE
added features page

### PR DESCRIPTION
**Pull request by AswaniBolisetti**
@ajay-dhangar 
I have features page and by clicking each feature that respective page will be opened
Kindly check it out

Could you please consider it as level 3 if you like it

https://github.com/user-attachments/assets/d9cea6e8-313c-4370-85f0-6a5bfd7e68b2

### CODE REVIEW

**Comment by ajay-dhangar** [comment]
{24815E48-942F-4F9A-AF98-386E3D5275B5}

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^5.3.0",
+    "react-router-dom": "^5.3.4",
```
Comment (ajay-dhangar) [inline package.json]: No need `"react-router-dom"` because we have already setup

**Comment by AswaniBolisetti** [comment]
@ajay-dhangar 
Actually I tried to solve them but i couldn't

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
don't install any packages

**Comment by ajay-dhangar** [comment]
I can close this issue today. However, we should not make this feature too complex. We need to maintain the codebase.

**Pull request closed without merging**